luigi/contrib/kubernetes.py

```diff
@@ -120,7 +94,7 @@ def kubernetes_namespace(self):
         For more details, please refer to:
         https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/
         """
-        return self.kubernetes_config.kubernetes_namespace
+        return "default"
```

Comment:
How do we set the `kubernetes_namespace` now? In the current implementation it is set [here](https://github.com/spotify/luigi/pull/3093/files#diff-00edfcf9a2971c757897a6fdb1a39816d7b5ada36e37a2315ed50003abde3d7eL209)

Reply:
@tnsetting This is a built-in feature in the foundational library we're using.  The config argument can be provided to specify the namespace.  See the `examples/kubernetes.py` for an example.

Reply:
The reason for "default" here, is that means as per how all of the kube utils work, if you don't specify anything you're in the "default" namespace by default.  With python, you can "override" this property, which is exactly what is happening in the examples/kubernetes.py

Reply:
So by this change, we're making it so you don't "have" to set a kubernetes_namespace, it'll run in default by default, unless you override this property.  Which is the same as you do before, basically.  This self.kubernetes_config concept in this retval doesn't make a ton of sense.  Kubernetes_config is/was replaced entirely, it was from the dated library.  This property is still overridable when you  use it.  Technically, your configs do not need to change at all it will function exactly as it did before.

Reply:
And technically, I could make this property not a function and just a property variable/value that you override.  I am happy to make that change for simplicity sake if desired?

Reply:
TL;DR: Everything works the same as it did before, you can override this property currently.  But, I can make this a property (variable) instead of a property (function) if desired for simplification.  I'll try to do that I think.

Reply:
Cleaned up code, simplified